math/integral_approximation.cpp

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file
+ * @brief Algorithm to compute integral approximation of function
+ *
+ * \author [Benjamin Walton](https://github.com/bwalton24)
+ * \author [Shiqi Sheng](https://github.com/shiqisheng00)
+ */
+#include <cassert>
+#include <cmath>
+#include <functional>
+#include <iostream>

```

Comment:
Add a one-line description of what the library/header is for (see the example below).
```c++
#include <cassert>    /// for assert
#include <iostream>  /// for IO operations
```